pkg/operator/sync.go

```diff
@@ -126,6 +144,69 @@ func (optr *Operator) syncTerminationHandler(config *OperatorConfig) error {
 	return nil
 }
 
+func (optr *Operator) syncWebhookConfiguration(config *OperatorConfig) error {
+	err := optr.syncValidatingWebhook(optr.kubeClient.AdmissionregistrationV1().ValidatingWebhookConfigurations(), mapiv1.WithValidatingHooks(config.TargetNamespace))
+	if err != nil {
+		return err
+	}
+	err = optr.syncMutatingWebhook(optr.kubeClient.AdmissionregistrationV1().MutatingWebhookConfigurations(), mapiv1.WithMutatingHooks(config.TargetNamespace))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (optr *Operator) syncValidatingWebhook(client admissionclientv1.ValidatingWebhookConfigurationInterface, hooks []admissionregistrationv1.ValidatingWebhook) error {
+	hook := mapiv1.NewValidatingWebhookConfiguration()
+	hook.Webhooks = hooks
```

Comment:
why is this broken down? why don't `NewValidatingWebhookConfiguration` just return the whole spec?